src/Workspaces/Core/Portable/Options/IWorkspaceOptionService.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Options
+{
+    /// <summary>
+    /// Interface used for exposing functionality from the option service that we don't want to 
```

Comment:
What i meant was "place to put internal functionality in case we make IOptionService public in the future."